Headset.desktop

```diff
@@ -0,0 +1,12 @@
+[Desktop Entry]
+Name=Headset
+Comment=Native YouTube Player
+GenericName=Headset
+Exec=Headset %U
+Icon=Headset
```

Comment:
Is it possible to use absolute path for the icon? from the readme - `/usr/lib/Headset/resources/app/icon.png`

Another question - is there a way to add a post  hook for the .deb package to copy and paste the file in the proper directory?  `/usr/share/applications/Headset.desktop`

I'm not a heavy Linux user so I just want to know if it's possible. That will solve the missing icon issue.